### PR TITLE
♻️ Type-fixing: registering a service should return undefined.

### PR DESCRIPTION
**Pull request by samouri**
Partial for https://github.com/ampproject/amphtml/issues/23582.

### CODE REVIEW

File: src/service/platform-impl.js [hunk 1]
```diff
@@ -230,8 +230,8 @@ export class Platform {
 
 /**
  * @param {!Window} window
- * @return {*} TODO(#23582): Specify return type
+ * @return {undefined}
```
Comment (dvoytenko) [inline src/service/platform-impl.js]: Just remove `@return` entirely?

Comment (samouri) [reply src/service/platform-impl.js]: That'll work! Just checked and the `jsdoc/require-returns` rule only requires a return if the function actually returns something.

**Comment by samouri** [comment]
Percy failed for reasons that...really don't seem like they could be related. I'll rerun the test to see if it was just a flake

**Comment by samouri** [comment]
@dvoytenko thanks for the review. I still don't have merge access, so whenever you have spare cycles could you submit this PR? Thanks!

**Pull request merged**